src/main/java/uk/gov/companieshouse/scanupondemand/orders/api/ScanUponDemandApiApplication.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.scanupondemand.orders.api;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ScanUponDemandApiApplication {
+
+	public static final String APPLICATION_NAMESPACE = "scan-upon-demand.orders.api.ch.gov.uk";
```

Comment:
Perhaps just keep the `LoggingUtils` constant?

Reply:
I followed the certified-copies approach which is different from other services approaches, which makes it a little bit confusing for me.

Reply:
Yes, well our approach is evolving over time. We just need one instance of this constant in the app. Observing that in some cases, this constant is used from within LoggingUtils methods, that might be the place for it. But at this point it shouldn't matter which, so long as we only define it once.

Reply:
Thanks for the explanation, I had a discussion in parallel with @hwilliams1 and now it clear to me. I will remove it and push it again as well as the other comment for the test class for the sake of good programming practices